pkg/controller/issueapi/logic.go

```diff
@@ -300,6 +300,6 @@ func (c *Controller) getAuthorizationFromContext(r *http.Request) (*database.Aut
 	return authorizedApp, currentUser, nil
 }
 
-func recordObservability(ctx *context.Context, result *issueResult) {
-	observability.RecordLatency(ctx, time.Now(), mLatencyMs, &result.obsBlame, &result.obsResult)
+func recordObservability(ctx context.Context, result *issueResult) {
```

Comment:
@yegle I think you said this needed to be a pointer for some reason, but I can't find the PR where you mentioned that.

Reply:
I'm going to assume we're trying to capture
`ctx = observability.WithRealmID(ctx, realm.ID)`
I'm not sure if that works - pointers to context is a bit of an odd pattern

Reply:
Actually I can't explain why we have `RecordLatency(ctx *context.Context` - it does not seem to really need the ctx pointer and doesn't modify it